esx_service/vmdk_ops.py

```diff
@@ -778,6 +778,7 @@ def disk_attach(vmdk_path, vm):
             msg=("Failed to add PVSCSI Controller: %s", ex.msg)
             return err(msg)
         # Find the controller just added
+        devices = vm.config.hardware.device
```

Comment:
Ah, I see. Can you please add this explanation to the comments ? Thanks.
LGTM after that.